prompter.ts

```diff
@@ -66,7 +66,7 @@ export class Prompter implements IPrompter {
 		return future;
 	}
 
-	public getPassword(prompt: string, options?: {allowEmpty?: boolean}): IFuture<string> {
+	public getPassword(prompt: string, options?: IAllowEmpty): IFuture<string> {
```

Comment:
I'd rather not have a default action here because I can't imagine a case where one'd want a default action to take if a password is not provided.
If, on the other hand, we use `IPrompterOptions` here and just don't respect the `defaultAction` that would be misleading to the coder.